### PR TITLE
Improved readme & added values to symfony/.env

### PR DESCRIPTION
**Pull request by Tapanila**
- Added instructions how to setup Tunkki when running it for the first time
- Added values so Tunkki compiles without errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,36 @@
...
+
+#### install assets
+* docker compose run --rm node yarn install
+* docker compose run --rm node yarn build
+
+### Access tunkki
+* open http://localhost:9090/ in your browser
+
+### Initial creation of new user and setting it as super admin
+* fill in the registration form at http://localhost:9090/profile/new and submit.
```
Comment (haivala) [inline README.md]: I already fixed the console command. It can be now used to create the user too.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,36 @@
...
+* Submission will result in error with default values since mattermost values are null
+* docker compose exec fpm ./bin/console entropy:member yourEmailFilledAtTheForm --super-admin
+
+### Setting up main website
+
+* login https://localhost:9090/login
```
Comment (haivala) [inline README.md]: s?

File: README.md (hunk 1 above)
Comment (haivala) [reply README.md]: https://github.com/EntropyRy/Tunkki/blob/symfony6/symfony/src/Command/UserCommand.php#L45

Comment (Tapanila) [reply README.md]: When I run
docker compose exec fpm ./bin/console entropy:member --create-user teemu@tapanila.net --super-admin

I get following errors:
In ExceptionConverter.php line 49:
                                                                                                                                        
  An exception occurred while executing a query: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test.member' doesn't exist  
                                                                                                                                        

In Exception.php line 28:
                                                                                         
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test.member' doesn't exist  
                                                                                         

In Statement.php line 130:
                                                                                         
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test.member' doesn't exist

Comment (Tapanila) [reply README.md]: Just something wrong with my setup. Nevermind this error

File: docker/fpm/dev.ini [hunk 3]
```diff
@@ -1,6 +1,6 @@
 date.timezone = Europe/Helsinki
 intl.default_locale = fi
-memory_limit = 256M
+memory_limit = 1024M
```
Comment (haivala) [inline docker/fpm/dev.ini]: Why? 256M is plenty

Comment (Tapanila) [reply docker/fpm/dev.ini]: These weren't supposed to be committed. 🥲

**Pull request merged**